tests/sentry/snuba/metrics/test_snql.py

```diff
@@ -278,41 +279,13 @@ def test_dist_count_aggregation_on_tx_status(self):
     def test_set_count_aggregation_on_tx_satisfaction(self):
         alias = "transaction.miserable_user"
 
-        assert miserable_users(self.org_id, self.metric_ids, alias) == Function(
-            "uniqIf",
-            [
-                Column("value"),
-                Function(
-                    "and",
-                    [
-                        Function(
-                            "equals",
-                            [
-                                Column(
-                                    resolve_tag_key(
-                                        UseCaseID.TRANSACTIONS,
-                                        self.org_id,
-                                        TransactionTagsKey.TRANSACTION_SATISFACTION.value,
-                                    )
-                                ),
-                                resolve_tag_value(
-                                    UseCaseID.TRANSACTIONS,
-                                    self.org_id,
-                                    TransactionSatisfactionTagValue.FRUSTRATED.value,
-                                ),
-                            ],
-                        ),
-                        Function(
-                            "in",
-                            [
-                                Column("metric_id"),
-                                list(self.metric_ids),
-                            ],
-                        ),
-                    ],
-                ),
-            ],
-            alias,
+        assert miserable_users(self.org_id, self.metric_ids, alias) == uniq_if_column_snql(
```

Comment:
Using `uniq_if_column_snql` in here helps me see differences with `on_demand_user_misery_snql_factory` since they both now use the same function.